src/pyatsimagebuilder/builder.py

```diff
@@ -269,12 +268,14 @@ def _process_repositories(self, repositories):
             assert not target.exists(), "%s already exists" % name
 
             # Clone and checkout the repo
-            git_clone(vals['url'], target, vals.get('commit_id', None), True)
+            git_clone(vals['url'], target, vals.get('commit_id', None), True,
+                      vals.pop('credentials', None), vals.pop('ssh_key', None))
```

Comment:
Maybe consider replacing the credentials/ssh_key with ****** instead of popping them? This way hides the fact that authentication was used at all.

Reply:
done